qumcmc/quantum_mcmc_routines_qulacs.py

```diff
@@ -205,7 +205,7 @@ def run_qc_quantum_step(
     J = model.get_J
 
     # init_qc=initialise_qc(n_spins=n_spins, bitstring='1'*n_spins)
-    gamma = np.round(np.random.uniform(0.25, 0.6), decimals=2)
+    gamma = np.round(np.random.uniform(0, 0.01), decimals=6)
```

Comment:
I would not hard code this small of a gamma. Can you free up this range as a tuple? 
`gamma_range = (0.25, 0.6)` as default. 
Then `np.round(np.random.uniform(*gamma_range ), decimals=6)`

Reply:
Yeah I had a similar idea, but do you want me to make the change in your branch right away? I was thinking to do that after I have made the merge !?